families/seth/src/sawtooth_seth/cli/account_import.go

```diff
@@ -56,7 +56,11 @@ func (args *AccountImport) Run(*Config) error {
 		return fmt.Errorf("Couldn't load key from file %v: %v", keyFilePath, err)
 	}
 
-	err = SaveKey(alias, string(buf), args.Force)
+	if keyFilePath[len(keyFilePath)-4:] == ".pem" {
```

Comment:
Can you use the ` path.Match("*.pem", keyname)` syntax here, like you use in `account_list.go`